src/components/cooperation-section-view/CooperationSectionView.tsx

```diff
@@ -0,0 +1,54 @@
+import Box from '@mui/material/Box'
+
+import { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import AppTextField from '~/components/app-text-field/AppTextField'
+import HeaderTextWithDropdown from '~/components/header-text-with-dropdown/HeaderTextWithDropdown'
+import ResourceItem from '~/containers/course-section/resource-item/ResourceItem'
+import { CourseSection, TextFieldVariantEnum } from '~/types'
+
+import { styles } from '~/components/cooperation-section-view/CooperationSectionView.styles'
+
+interface CooperationSectionViewProps {
+  id: string
+  item: CourseSection
+}
+
+const CooperationSectionView = ({ item, id }: CooperationSectionViewProps) => {
+  const [isVisible, setIsVisible] = useState<boolean>(true)
```

Comment:
Using generic here is redundant because here typescript can infer type of state variable and it's setter from initial value you passed into `useState`

Reply:
fixed